synthetics/test-data.json

```diff
@@ -0,0 +1,958 @@
+{
+  "Tests": [
+    {
+      "id": "1",
+      "name": "github.com ip test",
+      "type": "ip-address",
```

Comment:
FYI, the correct type is actually `ip`. Production API returns `ip-address`, but fix is already staged in Kentik's test environment.